content/general/concepts/number/number.md

```diff
@@ -0,0 +1,15 @@
+---
+Title: 'Number'
+Description: 'The number data type. It holds a number as its value.'
```

Comment:
Let's try to keep the meta-`Description` down to one sentence, maybe some like "The number data type encompasses many numeric values"?

Reply:
I wasn't quite sure what to put for it. So I just put that.